apiserver/facades/agent/secretsmanager/secrets_test.go

```diff
@@ -471,16 +471,15 @@ func (s *SecretsManagerSuite) TestUpdateSecretDuplicateLabel(c *gc.C) {
 	})
 }
 
+// TestRemoveSecrets tests that the facade returns without error when called to remove all revisions of a secret.
 func (s *SecretsManagerSuite) TestRemoveSecrets(c *gc.C) {
 	defer s.setup(c).Finish()
 
 	uri := coresecrets.NewURI()
 	expectURI := *uri
 	s.secretsState.EXPECT().GetSecret(&expectURI).Return(&coresecrets.SecretMetadata{}, nil)
-	s.secretsState.EXPECT().GetSecretRevision(&expectURI, 666).Return(&coresecrets.SecretRevisionMetadata{
-		Revision: 666,
-		ValueRef: &coresecrets.ValueRef{BackendID: "backend-id", RevisionID: "rev-666"},
-	}, nil)
+	// DeleteSecret returns the deleted external revisions iff it is deleting all revisions in a secret, so returning
+	// (not nil, nil) simulates that event.
```

Comment:
We no longer call `GetSecretRevision` due to the changes in `common/secrets/secrets.go`

Also added comments here as a driveby.  Not strictly needed, but felt like it would help the next reader.